src/modules/partition/core/PartitionActions.cpp

```diff
@@ -109,6 +109,10 @@ doAutopartition( PartitionCoreModule* core, Device* dev, Choices::AutoPartitionO
         partType = isEfi ? PartitionTable::gpt : PartitionTable::msdos;
     }
 
+    FileSystem::Type type;
+    PartUtils::findFS(o.defaultFsType, &type);
+    core->initLayout( type == FileSystem::Unknown ? FileSystem::Ext4 : type);
```

Comment:
I think the fallback filesystem is addressed here ...

Reply:
It's for the case when fs string in config is invalid